File: src/tests/Markify.Fixtures/Projects/Source/Struct/StructPartial.cs
using System;
using System.Collections;

public partial struct FooType : IDisposable
{
    private int _fieldOne;

    private int PropertyOne { get; set; }

    public event EventHandler Done;

    partial void PartialMethod();

    public void MethodOne(int foo);
}

public partial struct FooType : IEnumerable
{
    private float _fieldTwo;

    private int PropertyTwo { get; set; }

    public event EventHandler Started;

    partial void PartialMethod() { }

    partial void PartialMethod(int foo);

    public void MethodTwo(int foo, int bar) { }
}

File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Struct/StructPartial.vb
Imports System
Imports System.Collections

Public Partial Structure FooType
    Implements IDisposable
    Private _fieldOne As Integer

    Private Property PropertyOne As Integer

    Public Event Done As EventHandler

    Partial Sub PartialMethod()
    End Sub

    Public Sub MethodOne(foo As Integer)
    End Sub
End Structure

Public Partial Structure FooType
    Implements IEnumerable
    Private _fieldTwo As Decimal

    Private Property PropertyTwo As Integer

    Public Event Started As EventHandler

    Partial Sub PartialMethod()
    End Sub

    Partial Sub PartialMethod(foo As Integer)
    End Sub

    Public Sub MethodTwo(foo As Integer, bar As Integer)
    End Sub
End Structure